### ISSUE TITLE
Unable to build from source on macOS

### ISSUE THREAD

**Issue by Rishav1** [tensorflow/federated#771]
Platform specs:
OS: macOS Mojave 10.14.5
Python: 3.7.4
Build: From Source(d02309a)

packages | version
--- | --- 
six|                           1.12.0
tb-nightly|                    2.1.0a20191013
tensorflow-model-optimization| 0.1.3
tensorflow-privacy|            0.1.0
tf-estimator-nightly|          2.0.0.dev2019101401
tf-nightly|                    2.1.0.dev20191014
tfa-nightly|                   0.7.0.dev20191010

Followed the instructions in [building from source](https://github.com/tensorflow/federated/blob/master/docs/install.md#build-the-tensorflow-federated-pip-package).

The following tests fail:
```
//tensorflow_federated/python/core/impl:intrinsic_bodies_test           TIMEOUT in 61.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/core/impl/intrinsic_bodies_test/test.log
//tensorflow_federated/python/core/utils:federated_aggregations_test    TIMEOUT in 61.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/core/utils/federated_aggregations_test/test.log
//tensorflow_federated/python/research/robust_aggregation:robust_federated_aggregation_test FAILED in 5.6s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/research/robust_aggregation/robust_federated_aggregation_test/test.log
//tensorflow_federated/python/research/simple_fedavg:simple_fedavg_test  FAILED in 5.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/research/simple_fedavg/simple_fedavg_test/test.log
//tensorflow_federated/python/research/utils:training_loops_test         FAILED in 5.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/research/utils/training_loops_test/test.log
//tensorflow_federated/python/simulation/datasets:emnist_test            FAILED in 4.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/simulation/datasets/emnist_test/test.log

Executed 89 out of 89 tests: 83 tests pass and 6 fail locally.
INFO: Build completed, 6 tests FAILED, 404 total actions
```

Hello World [snippet](https://github.com/tensorflow/federated/blob/master/docs/install.md#10-test-tensorflow-federated) also fails.
```
(venv)  rishavchourasia@Rishavs-MacBook-Pro  /tmp/project  python -c "import tensorflow_federated as tff; print(tff.federated_computation(lambda: 'Hello World')())"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_federated/__init__.py", line 62, in <module>
    from tensorflow_federated.python import simulation
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_federated/python/simulation/__init__.py", line 23, in <module>
    from tensorflow_federated.python.simulation import datasets
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_federated/python/simulation/datasets/__init__.py", line 21, in <module>
    from tensorflow_federated.python.simulation.datasets import emnist
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_federated/python/simulation/datasets/emnist.py", line 29, in <module>
    import tensorflow_addons.image as tfa_image
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/__init__.py", line 21, in <module>
    from tensorflow_addons import activations
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/activations/__init__.py", line 21, in <module>
    from tensorflow_addons.activations.gelu import gelu
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/activations/gelu.py", line 25, in <module>
    get_path_to_datafile("custom_ops/activations/_activation_ops.so"))
  File "/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_core/python/framework/load_library.py", line 57, in load_op_library
    lib_handle = py_tf.TF_LoadLibrary(library_filename)
tensorflow.python.framework.errors_impl.NotFoundError: dlopen(/private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/custom_ops/activations/_activation_ops.so, 6): Symbol not found: __ZN10tensorflow11GetNodeAttrERKNS_9AttrSliceEN4absl14lts_2019_08_0811string_viewEPb
  Referenced from: /private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/custom_ops/activations/_activation_ops.so
  Expected in: /private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_core/python/../libtensorflow_framework.2.dylib
 in /private/tmp/project/venv/lib/python3.7/site-packages/tensorflow_addons/custom_ops/activations/_activation_ops.so
```

**Comment by ZacharyGarrett** [comment]
Hi @Rishav1,

We have seen issues with some of the earlier `tf-nightly` and `tfa-nightly` versions. Would you be up for trying to see if updating to a newer version has the same issue?

We have seen `tfa-nightly-0.7.0.dev20191017` but `tfa-nightly-0.7.0.dev20191010` has failed in the fast.

**Comment by Rishav1** [comment]
Issue seems to be resolved in `tfa-nightly-0.7.0.dev20191031`. All bazel tests also passed except a single test timing out.

```
//tensorflow_federated/python/core/impl:intrinsic_bodies_test           TIMEOUT in 61.0s
  /private/var/tmp/_bazel_rishavchourasia/26093410bba57464b779c2b8c15ff77a/execroot/org_tensorflow_federated/bazel-out/darwin-opt/testlogs/tensorflow_federated/python/core/impl/intrinsic_bodies_test/test.log

Executed 89 out of 89 tests: 88 tests pass and 1 fails locally.
INFO: Build completed, 1 test FAILED, 404 total actions
```
